lib/tenstorrent/bh_arc/gddr.c

```diff
@@ -26,9 +26,12 @@
 #include <zephyr/devicetree.h>
 #include <zephyr/drivers/clock_control/clock_control_tt_bh.h>
 #include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/dma.h>
+#include <zephyr/drivers/dma/dma_tt_bh_noc.h>
 
 static const struct device *const pll_dev_3 = DEVICE_DT_GET_OR_NULL(DT_NODELABEL(pll3));
 static const struct device *flash = DEVICE_DT_GET_OR_NULL(DT_NODELABEL(spi_flash));
+static const struct device *const dma_dev = DEVICE_DT_GET_OR_NULL(DT_NODELABEL(dma1));
```

Comment:
It would be good to have a slightly more descriptive name here, because both the ARC DMA and NOC DMA are used for MRISC and ERISC IIRC.
```suggestion
static const struct device *const noc_dma_dev = DEVICE_DT_GET_OR_NULL(DT_NODELABEL(dma1));
```